packages/zosjobs/src/cli/submit/stdin/stdin.definition.ts

```diff
@@ -27,6 +27,17 @@ export const StdinDefinition: ICommandDefinition = {
                 " If you use this option you will wait the job to complete.",
             type: "boolean"
         },
+        {
+            name: "wait-for-output", aliases: ["wfo"],
```

Comment:
Should this have a `conflictsWith` property as well? (for `vasc` and `directory`)?

Reply:
Same for the other definitions on the other submit commands.

Reply:
I'm not sure actually. --directory and --vasc both involve waiting for output so in a way they don't conflict, right? Or am I misunderstanding?

Reply:
Yeah, sorry not wait for output, but wait for active?

Reply:
Oh, yes you're right